Documentation/policies/package-servicing.md

```diff
@@ -0,0 +1,50 @@
+# .NET Package Servicing Model
+
+Most packages in the .NET ecosystem support only the latest version. Many packages from the .NET team operate on this same model. A subset of our packages operate on another model where multiple version bands are supported at the same time, matching in-support runtimes, offering more flexibility to users.
+
+Users should aim to maintain their package references to be as close to the supported version as possible. Being close to a supported version means that you have most or all security fixes and likely have no latent compatibility risk that can be present when there is a big gap between your reference and the supported version.
+
+## Latest version servicing
+
+The default package servicing model is “latest version”. The team publishes an updated package with a security and/or functionality fix. Users can upgrade to the new latest version to get the fix.
+
+The following libraries use this servicing model:
+
+- [.NET Aspire](https://github.com/dotnet/aspire)
+- [ML.NET](https://github.com/dotnet/machinelearning)
+- [Maintenance-packages](https://github.com/dotnet/maintenance-packages)
+
+## Runtime-band servicing
+
+Some packages match a library (in name and content) in the runtime. In this case, the team publishes  fixes for the library in the runtime and the package at the same time. The version numbers will match. Users can upgrade to the new latest patch version to get the fix.
+
+Users can upgrade to the latest library version (like 8.0.x) that matches a supported runtime (like 8.0). Users can also move to the latest patch of a newer major version (like 9.0.x) provided that the runtime is still in support.
+
+The advantage of matching versions is that the runtime library will be used in place of the package, resulting in smaller deployments and servicing being provided by the runtime. The advantage of using a newer package version is getting access to newer functionality.
+
+Apps only ever need a package reference to one of these libraries to use a newer major version. It is more common for libraries to have a reference, particularly .NET Standard libraries.

```

Comment:
This statement was confusing had to read it many times to understand. Maybe being more explicit about the user writing a library (particularly with a .NET Standard target) will have explicit package references.

Reply:
I agree that it is confusing. Better?

```suggestion
It common for .NET Standard libraries to have a reference to these libraries. Libraries and app projects only need a reference if they need access to newer features in a later major version (than their target framework exposes).
```